docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,628 @@
+---
+title: UID2 Client-Side Integration Guide for Mobile
+sidebar_label: Client-Side Integration for Mobile
+pagination_label: UID2 Client-Side Integration Guide for Mobile
+description: Information about setting up a client-side mobile integration.
+hide_table_of_contents: false
+sidebar_position: 04
+displayed_sidebar: sidebarPublishers
+---
+
+import Link from '@docusaurus/Link';
+
+# UID2 Client-Side Integration Guide for Mobile
+
+(**GWH__SW: audience? Is this only for publishers, or possibly advertisers also? (advertisers with apps?)**)
```

Comment:
(App) Publishers only for now. Advertiser may be relevant in the future for pixel use cases https://unifiedid.com/docs/sharing/sharing-tokenized-from-data-pixel